libs/webb-ui-components/src/components/BottomDialog/BottomDialog.tsx

```diff
@@ -0,0 +1,10 @@
+import { forwardRef } from 'react';
+import * as Dialog from '@radix-ui/react-dialog';
+
+import { BottomDialogProps } from './types';
+
+export const BottomDialog = forwardRef<HTMLDivElement, BottomDialogProps>(
+  (props, _) => {
```

Comment:
You should pass the `ref` here to the child component instead of `_`, if not, we will get a warning in the browser console.